pkl-tools/pkl-tools.gradle.kts

```diff
@@ -65,6 +65,58 @@ val javadocJar by
 
 tasks.shadowJar { archiveBaseName.set("pkl-tools-all") }
 
+private fun Exec.configureTestStartFatJar(launcher: Provider<JavaLauncher>) {
+  dependsOn(tasks.shadowJar)
+  group = "verification"
+
+  // placeholder output to satisfy up-to-date check
+  val outputFile = layout.buildDirectory.file("testStartFatJar/${name}.txt")
```

Comment:
Don't you have to call `outputs(outputFile)` as well? 🤔

Reply:
Yes, you're right!